feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.component.ts

```diff
@@ -40,22 +40,29 @@ export class ConfiguratorAttributeQuantityComponent
 
   ngOnInit() {
     this.quantity.setValue(this.quantityOptions?.initialQuantity?.quantity);
-
-    if (this.quantityOptions?.disableQuantityActions) {
-      this.quantity.disable();
-    }
+    const sub = this.quantityOptions.disableQuantityActions?.subscribe(
```

Comment:
Why is the needed? Can't we directly subscribe on quantityOptions.disableQuantityActions in the view?
(via readOnly attribute?)

Reply:
it was already like this - but i quickly tested it. When using the disabled property, the styling is different. Plus and minus buttons just disappear. When disabling/enabling method, plus and minus get greyed out. I guess the later is, what we desire.

Reply:
Ok I see